docs/skopeo.1.md

```diff
@@ -64,6 +64,11 @@ Uses the system's trust policy to validate images, rejects images not trusted by
 
   _destination-image_ use the "image name" format described above
 
+  **--authfile** _path_
+
+  Path of the authentication file. Default is ${XDG_RUNTIME_DIR}/containers/auth.json
+  which is set using `kpod login` or `docker login`
```

Comment:
This seems to say `docker login` writes to `auth.json`, which is not true. (Also in two other instances.)

Reply:
If we have this in kpod and/or Buildah, can we touch it up there too please?

Reply:
fixed.
@TomSweeneyRedHat there is no mention of docker login in buildah or kpod. Is it something we want to add?